slam_toolbox/include/slam_toolbox/get_pose_helper.hpp

```diff
@@ -37,8 +37,15 @@ class GetPoseHelper
   {
   };
 
-  bool getOdomPose(karto::Pose2& karto_pose, const ros::Time& t)
+  bool getOdomPose(karto::Pose2& karto_pose,
+    const ros::Time& t,
+    std::string ref_frame = std::string(""))
```

Comment:
Is it sensible to have a default value here? Is this used as default?

Reply:
Removed default value for ref_frame